src/wp-includes/capabilities.php

```diff
@@ -874,6 +874,9 @@ function map_meta_cap( $cap, $user_id, ...$args ) {
  *              passed, whether the current user has the given meta capability for the given object.
  */
 function current_user_can( $capability, ...$args ) {
+	if ( ! function_exists( 'wp_get_current_user' ) ) {
+		return false;
```

Comment:
This should not return false because it means user can't and maybe user can but we don't know yet if they can because we can't wp_get_current_user. 😆 It should return an error that can be used so that the extender or core calls `current_user_can` correctly.